scripts/monte_carlo.py

```diff
@@ -180,47 +363,49 @@ def monte_carlo_sampling_scipy(
     ### SCENARIO INPUTS
     ###
     MONTE_CARLO_PYPSA_FEATURES = {
-        k: v for k, v in monte_carlo_config["pypsa_standard"].items() if v
+        k: v.get("scale") for k, v in monte_carlo_config["uncertainties"].items() if v
```

Comment:
do we have scale?
I think we have same legacy to fix here

Reply:
Haa...No we no longer have scale.